src/app/pages/MapPage/detail/DetailTable.tsx

```diff
@@ -1,35 +1,43 @@
 import { FunctionComponent } from 'react'
 import styled from 'styled-components'
+import { Paragraph } from '@amsterdam/asc-ui'
 import { DetailResultItemTable } from '../../../../map/types/details'
 import { Table, TableData, TableHeader, TableRow } from '../../../components/Table'
 
 const TableWrapper = styled.div`
   width: 100%;
-  overflow-x: scroll;
+  overflow-x: auto;
 `
 
 export interface DetailTableProps {
   item: DetailResultItemTable
 }
 
-const DetailTable: FunctionComponent<DetailTableProps> = ({ item }) => (
-  <TableWrapper data-testid="detail-table">
-    <Table>
-      <TableRow header>
-        {item.headings.map((heading) => (
-          <TableHeader key={heading.key}>{heading.title}</TableHeader>
-        ))}
-      </TableRow>
-      {item.values?.map((value, index) => (
-        // eslint-disable-next-line react/no-array-index-key
-        <TableRow key={index}>
-          {item.headings.map((heading) => (
-            <TableData key={heading.key}>{value[heading.key]}</TableData>
+const DetailTable: FunctionComponent<DetailTableProps> = ({ item }) =>
+  item.values?.length ? (
+    <TableWrapper data-testid="detail-table">
+      <Table>
+        <thead>
+          <TableRow header>
+            {item.headings.map(({ key, title }) => (
+              <TableHeader key={key}>{title}</TableHeader>
+            ))}
+          </TableRow>
+        </thead>
+        <tbody>
+          {item.values?.map((value, index) => (
```

Comment:
No need to do a check on the values again, this is already happening before.
```suggestion
          {item.values.map((value, index) => (
```